components/tekton/pipelineruns/run-sync-test/run-test-suite.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: run-vse-sync-test-run
+  labels:
+    tekton.dev/pipeline: run-vse-sync-test
+spec:
+  serviceAccountName: pipeline-credentials-sa
+  # serviceAccountName: robot-docker-basic
+  params:
+    - name: kubeconfig-secret
+      value: falcon-admin-kubeconfig
+    - name: kubeconfig-namespace
+      value: falcon
+    - name: kubeconfig-secret-dest
+      value: falcon-admin-kubeconfig
+  pipelineRef:
+    name: run-vse-sync-test
+  workspaces:
+    - name: report
+      volumeClaimTemplate:
+        spec:
+          accessModes:
+            - ReadWriteOnce
+          resources:
+            requests:
+              storage: 10Gi
```

Comment:
Would be interesting to understand how much storage is required in the long run, as 10Gi seems like a lot on paper with examples in tekton.dev showing 16Mb in examples.

Reply:
Good point this is way too much and not needed. This is leftover from the previous test pipeline and is not needed anymore since we take the test output.
I removed the use of workspace